### PR TITLE
[JENKINS-32777] Cover 'delete-builds' CLI by tests

### PR DESCRIPTION
**Pull request by pjanouse**
Please review, thx. Pavel

### CODE REVIEW

File: test/src/test/java/hudson/cli/DeleteBuildsCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+        final CLICommandInvoker.Result result = command
+                .authorizedTo(Jenkins.READ, Job.READ, Run.DELETE)
+                .invokeWithArgs("aProject", "0");
+        assertThat(result, succeeded());
+        assertThat(result.stdout(), containsString("Deleted 0 builds"));
+    }
+
+    @Test public void deleteBuildsShouldSuccessEvenTheBuildIsRunning() throws Exception {
+        FreeStyleProject project = j.createFreeStyleProject("aProject");
+        project.getBuildersList().add(new Shell("echo 1\nsleep 10s"));
```
Comment (olivergondza) [inline test/src/test/java/hudson/cli/DeleteBuildsCommandTest.java]: This is fragile. Something like `ExecutorTest.BlockingBuilder` should be used.

**Comment by daniel-beck** [comment]
LGTM

File: test/src/test/java/hudson/cli/DeleteBuildsCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+                .authorizedTo(Jenkins.READ, Job.READ, Run.DELETE)
+                .invokeWithArgs("aProject", "2-3");
+        assertThat(result, succeeded());
+        assertThat(result.stdout(), containsString("Deleted 1 builds"));
+        assertThat(((FreeStyleProject) j.jenkins.getItem("aProject")).getBuilds().size(), equalTo(0));
+    }
+
+    @Test public void deleteBuildsManyShouldSuccessEvenTheFirstAndLastBuildDoesNotExist() throws Exception {
+        FreeStyleProject project = j.createFreeStyleProject("aProject");
+        project.scheduleBuild2(0).get();
```
Comment (oleg-nenashev) [inline test/src/test/java/hudson/cli/DeleteBuildsCommandTest.java]: j.buildAndAssertSuccess would hide some implementation details. Not important

**Comment by oleg-nenashev** [comment]
:+1:

**Comment by oleg-nenashev** [comment]
@olivergondza Regarding your comment... Do you want to get it fixed before the merge?

**Comment by pjanouse** [comment]
@oleg-nenashev Yes, I'll fix it before merge.

**Comment by oleg-nenashev** [comment]
👍

**Pull request merged**